ITMO_FS/filters/multivariate/DISRwithMassive.py

```diff
@@ -54,30 +54,42 @@ def __init__(self, expected_size=None):
         self.selected_features = None
 
     def __count_weight(self, i):
-        return 2 * self._vertices[i] * np.multiply(self._edges[i], self._vertices)
-
-    def run(self, X, y):
+        temp_difference = 0
+        for j in range(self.n_features):
```

Comment:
this could be done with numpy multiply

Reply:
wrapping the whole optimization in np.sum() seems to give the same values as the non-optimized version, changed